test/compare_against_shared.cu

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
```

Comment:
Can you add some description of what this test does and how does it work? I think `compare_against_shared.cu` filename is a bit confusing, what is `shared` here, what are we comparing against? Probably need to rename this file to make it more clear.

Reply:
The description is added in https://github.com/rapidsai/distributed-join/pull/43/commits/e6c37662ed5d23087d6491a33563211cc2e4375b, and the file is renamed to `compare_against_single_gpu.cu`.